polkadot/scripts/packaging/rpm-maintainer-scripts/rpm-postinst.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+# Post-install script for RPM package
+
+set -e
+
+config_file="/etc/default/polkadot"
+
+# Create polkadot group if it doesn't exist
+getent group polkadot >/dev/null || groupadd -r polkadot
+
+# Create polkadot user if it doesn't exist
+getent passwd polkadot >/dev/null || \
+    useradd -r -g polkadot -d /home/polkadot -m -s /sbin/nologin \
+    -c "User account for running polkadot as a service" polkadot
+
+# Create default config file if it doesn't exist
+if [ ! -e "$config_file" ]; then
+    echo 'POLKADOT_CLI_ARGS=""' > "$config_file"
+fi
+
+# Set correct permissions for binaries and service files
+echo "Setting file permissions..."
+chmod 755 /usr/bin/polkadot || true
+chmod 755 /usr/lib/polkadot || true
+chmod 755 /usr/lib/polkadot/* || true
+chmod 644 /usr/lib/systemd/system/polkadot.service || true
```

Comment:
Why? That should be handled by the `%install` section of the RPM spec, e.g. `install -m 0755 %{name} %{buildroot}%{_bindir}/%{name}` etc. If the permissions in the spec are not correct and they are set up in the post-install script, the filesystem state will diverge with the RPM state (i.e RPM will show that the file contents are changed in the fs with respect to the package version, which is not right)